Insert a placeholder that MacPorts will replace with its CFLAGS since
this configure script overwrites the CFLAGS environment variable.
https://github.com/nh13/BFAST/issues/5
--- configure.orig	2011-10-29 14:32:59.000000000 -0500
+++ configure	2022-01-29 02:24:29.000000000 -0600
@@ -3581,7 +3581,7 @@
 
 
 # set CFLAGS and CXXFLAGS
-default_CFLAGS="-Wall -g -O2 -pthread";
+default_CFLAGS="-Wall -g @CFLAGS@ -pthread";
 extended_CFLAGS="";# "-m64 -D_FILE_OFFSET_BITS=64";
 
 # Define some variables
